src/herder/test/HerderTests.cpp

```diff
@@ -40,6 +40,10 @@ TEST_CASE("standalone", "[herder]")
     cfg.QUORUM_SET.validators.clear();
     cfg.QUORUM_SET.validators.push_back(v0NodeID);
 
+    // Do our setup in version 1 so that for_all_versions below does not
+    // try to downgrade us from >1 to 1.
+    cfg.LEDGER_PROTOCOL_VERSION = 1;
```

Comment:
only the section ` SECTION("txset with valid txs - but failing later")` seems to be doing that, this breaks the other section(s) that will only test old behavior.
I would suggest doing something similar that I did in the CAP0005 PR, where I extracted the setup code into a function of its own that takes a protocol version as a parameter (and create the app with the desired version)

Reply:
fixed